python/GafferTest/TestRunner.py

```diff
@@ -209,7 +209,8 @@ def addSuccess( self, test ) :
 				new = min( timings )
 				old = min( test.previousTimings )
 				reduction = 100 * (old-new)/old
-				if reduction > 2 :
+				#if reduction > 2 :
+				if True:
```

Comment:
There is some concept of verbosity in Python, but it's not immediately clear to me whether it would actually be appropriate to define new meanings for it, since it has a pretty defined meaning, and I'm not sure exactly sure how we would set it, and it seems like we might need to store a separate value ourselves anyway, since there doesn't seem to be any way to read the verbosity back from a TextTestRunner after we set it?

For the moment, I have simply omitted this - it's not necessary to get this PR merged, though in the long term, we probably do need to do something like this to make this sort of performance test more useful.